build.sh

```diff
@@ -1,32 +1,41 @@
-#!/bin/bash
+#!/usr/bin/env bash
+# shellcheck disable=2086
 
-git clone https://github.com/perkinslr/buildroot-sedna && \
-git clone https://github.com/perkinslr/buildroot --depth=1 && \
+git clone https://github.com/OpenComputers2-Reimagined/buildroot-sedna && \
+git clone https://github.com/OpenComputers2-Reimagined/buildroot --depth=1 && \
 
-cp buildroot-sedna/config buildroot/.config && \
-cp buildroot-sedna/linuxconfig buildroot && \
 pushd buildroot || exit 1
 
-export JOBS="-j$(python -c 'print(__import__("os").cpu_count() * 3 // 4)') -l$(python -c 'print(__import__("os").cpu_count() * 3 // 4)')"
 
-make $JOBS linux-configure  || exit 1 
-mv linuxconfig ./output/build/linux-6*/.config || exit 1
```

Comment:
Why was the kernel config copied this way by the way?  I assume you just didn't know where the config file buildroot uses is stored but just making sure.